src/components/Header/Heading.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import getElementType from '../../lib/getElementType';
+import './Heading.scss';
+
+const Heading = (props) => {
+  const {
+    className,
+    children,
+    size,
+  } = props;
+
+  const Element = getElementType(Heading, props);
+
+  const classes = classNames(
+    className,
+    'heading',
+    { [`heading--${size}`]: size },
+  );
+
+  return (
+    <Element className={classes}>{children}</Element>
+  );
+}
+
+Heading.propTypes = {
+  as: PropTypes.elementType,
```

Comment:
should this be more specific? `[div, h1, h2, h3, h4, h5, h6]`

Reply:
For this component I'd only like to take header values, but that function was crafted as a common utility to use elsewhere as needed so it includes that div fallback.